doc/modules/ensemble.rst

```diff
@@ -1,534 +1,437 @@
+`categorical_features` parameter, indicating which feature is categorical. In
+the following, the first feature will be treated as categorical and the
+second feature as numerical::
 
-Parameters
-----------
+  >>> gbdt = HistGradientBoostingClassifier(categorical_features=[True, False])
 
-The main parameters to adjust when using these methods is ``n_estimators`` and
-``max_features``. The former is the number of trees in the forest. The larger
-the better, but also the longer it will take to compute. In addition, note that
-results will stop getting significantly better beyond a critical number of
-trees. The latter is the size of the random subsets of features to consider
-when splitting a node. The lower the greater the reduction of variance, but
-also the greater the increase in bias. Empirical good default values are
-``max_features=1.0`` or equivalently ``max_features=None`` (always considering
-all features instead of a random subset) for regression problems, and
-``max_features="sqrt"`` (using a random subset of size ``sqrt(n_features)``)
-for classification tasks (where ``n_features`` is the number of features in
-the data). The default value of ``max_features=1.0`` is equivalent to bagged
-trees and more randomness can be achieved by setting smaller values (e.g. 0.3
-is a typical default in the literature). Good results are often achieved when
-setting ``max_depth=None`` in combination with ``min_samples_split=2`` (i.e.,
-when fully developing the trees). Bear in mind though that these values are
-usually not optimal, and might result in models that consume a lot of RAM.
-The best parameter values should always be cross-validated. In addition, note
-that in random forests, bootstrap samples are used by default
-(``bootstrap=True``) while the default strategy for extra-trees is to use the
-whole dataset (``bootstrap=False``). When using bootstrap sampling the
-generalization error can be estimated on the left out or out-of-bag samples.
-This can be enabled by setting ``oob_score=True``.
+Equivalently, one can pass a list of integers indicating the indices of the
+categorical features::
 
-.. note::
+  >>> gbdt = HistGradientBoostingClassifier(categorical_features=[0])
 
-    The size of the model with the default parameters is :math:`O( M * N * log (N) )`,
-    where :math:`M` is the number of trees and :math:`N` is the number of samples.
-    In order to reduce the size of the model, you can change these parameters:
-    ``min_samples_split``, ``max_leaf_nodes``, ``max_depth`` and ``min_samples_leaf``.
+The cardinality of each categorical feature should be less than the `max_bins`
+parameter, and each categorical feature is expected to be encoded in
+`[0, max_bins - 1]`. To that end, it might be useful to pre-process the data
+with an :class:`~sklearn.preprocessing.OrdinalEncoder` as done in
+:ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_categorical.py`.
```

Comment:
I thought we merged a PR that makes it so that all the rare categories above 255 are merged into one (the last) bin?